docs/demos/demos/intro_demo.py

```diff
@@ -0,0 +1,256 @@
+# methods.
+
+circuit = Circuit(num_qubits=2, num_bits=2)
+
+###############################################################################
+# You can use the `operations` module (shortened above to `ops`) to access a
+# variety of quantum gates; for example, a Pauli X operator.
+
+ops.X()
+
+###############################################################################
+# Notice above that an operation can be instantiated without declaring which
+# qubits it should be applied to.
+#
+# The matrix property can be accessed either via the operation class itself or an
+# instance of the operation, which additionally can contain parameters and qubit
+# information.
+
+ops.X.matrix
+
+###############################################################################
+# If the matrix representation is dependent on parameters (e.g., the rotation operation `ops.RX`)
+# it can only be retrieved from an instance.
+
+ops.RZ(4.2).matrix
+
+###############################################################################
+# ## The circuit context
+#
+# Operations are applied by calling either a class or an instance of a class
+# within the context of the circuit. 
+#
+# ```python
+# with circuit.context:
+#     # apply operations here
+# ```
+#
+
+###############################################################################
+# When activating the context, a named tuple containing reference registers to the
+# circuit's qubits and classical bits is returned. You can also access the qubit
+# registers directly via the `Circuit.qregisters` property, or the reference
+# registers containing all the qubits via `Circuit.qubits`.
+#
+# In the example below, the circuit contains a single qubit register with two
+# qubits; it could contain any number of qubit registers. You can
+#
+# * add another register with the `Circuit.add_qregister` method, where an argument `n`
+#   is the number of qubits in the new register
+# * add a qubit with `Circuit.add_qubit`, optionally passing a qubit object
+#   and/or a register to which to add it.
+
+circuit = Circuit(2)
+
+with circuit.context as reg:
+    ops.X(reg.q[0])
+
+###############################################################################
+# This example created a circuit object with two qubits in its register, applying
+# a single X gate to the first qubit. Print the circuit to see general information
+# about it: type of circuit, number of qubits/bits, and number of operations.
+
+print(circuit)
+
+###############################################################################
+# ## Applying gates to circuits
+#
+# You can apply operations to a circuit in several different ways, as demonstrated
+# in the example below. You can pass both qubits and parameters as either single
+# values (when supported by the gate) or sequences. Note that different types of
+# gates accept slightly different arguments, although you can _always_ pass the
+# qubits as sequences via the keyword argument `qubits`.
+#
+# :::note
```

Comment:
Not working:
![image](https://github.com/dwavesystems/dwave-gate/assets/34041130/0f5691c8-adcc-412c-a0aa-c547c2ce0d9e)

Reply:
Wonder why it seemingly works for me but not for you. How are you building the docs? :thinking:
![image](https://github.com/dwavesystems/dwave-gate/assets/6934626/7c610f02-076b-4b3c-9852-a0acc32d7cb0)

Reply:
```
make demos
make html
```

Reply:
Are you looking at the rendered docs (i.e., the rst file) or the jupyter notebook? The latter will not render the notebox, but the former should be displayed as above, rendering the docs nicely.

Reply:
By "rendered docs" I mean the HTML. I get the same whether I use ``make html`` or I do "sphinx-build -b html`` after running ``make demos``.

![image](https://github.com/dwavesystems/dwave-gate/assets/34041130/fd4ef6a5-c73f-4061-9815-ae483affe05f)

If you don't see this too, I will try looking into it either tomorrow or early next week

Reply:
Strange. @JoelPasvolsky seemed to get it to work when building everything in Codespaces and suspected it had to do with his Ubuntu version. Since it seems to be causing issues I'll look into it, but as long as the demos are built correctly in the live docs I'd think it's fine.

Reply:
I recently updated my Ubuntu so if it's needed to help debug I can run locally again and see if it now works